src/commands/sns_summary.rs

```diff
@@ -0,0 +1,31 @@
+use crate::commands::sns_canisters_summary::SnsCanistersSummaryOps;
+use crate::commands::{nervous_system_parameters, sns_canisters_summary};
+use crate::lib::signing::IngressWithRequestId;
+use crate::lib::AnyhowResult;
+use crate::SnsCanisterIds;
+use clap::Parser;
+
+#[derive(Parser, Debug)]
+#[clap(about, version, author)]
```

Comment:
Same here: This probably should go in mod.rs on the top level command instead on only some of the sub commands. If you feel strongly about it, please move it there. If not please remove it.